i18n/ads-language-pack-de/package.json

```diff
@@ -11,7 +11,7 @@
 	"license": "SEE SOURCE EULA LICENSE IN LICENSE.txt",
 	"engines": {
 		"vscode": "*",
-		"azdata": ">=1.29.0"
+		"azdata": "^0.0.0"
```

Comment:
These should be >=1.31.0 right?

Reply:
That is correct, I'm changing it to ^1.31.0 since that's following what vscode uses.

Reply:
Is it safe to create a main build then use the resulting vsixes from there to overwrite the ones in storage explorer or will this require a version bump?

Reply:
Why would we need to release a new version? Go ahead and change the engine string if you want, but that kind of change doesn't have any impact so I don't think we need to update the VSIXs that are published.

Reply:
It doesn't affect the fact you can install the langpack, just that some strings might not work on older versions.